src/System.Net.Requests/tests/HttpWebResponseHeaderTest.cs

```diff
@@ -96,23 +95,34 @@ public void HttpContinueMethod(int StatusCode, WebHeaderCollection httpHeaders)
                 WebResponse response = await getResponse;
                 HttpWebResponse httpResponse = (HttpWebResponse)response;
                 httpResponse.Close();
-                Assert.Throws<ObjectDisposedException>(() =>
+                if (PlatformDetection.IsFullFramework)
```

Comment:
Is the response Stream usable in this state in desktop?  If so, seems like one we should fix in core.